TestProjects/HDRP_Tests/ProjectSettings/ProjectSettings.asset

```diff
@@ -153,7 +153,7 @@ PlayerSettings:
     enable360StereoCapture: 0
   isWsaHolographicRemotingEnabled: 0
   protectGraphicsMemory: 0
-  enableFrameTimingStats: 0
+  enableFrameTimingStats: 1
```

Comment:
could it slow down our tests? if so, is there a risk of timeout?

Reply:
It shouldn't. When this flag is enabled, it will affect master builds too - it enables a some frame-level timing stats on the backend. These are the timings that could be used to implement dynamic resolution scaling logic. It's not zero overhead but should be very small. @bandures might have some numbers on this.